### PR TITLE
HPCC-31506 WuSnapShot change in casing

### PR DESCRIPTION
**Pull request by kunalaswani**
WuSnapshot to be replaced by WuSnapShot in translations files as well as the name change to be made in QuerySetDetailsWidget.html.

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: esp/src/src/nls/hpcc.ts [hunk 1]
```diff
@@ -1130,7 +1130,7 @@ export = {
         WSDL: "WSDL",
         WUID: "WUID",
         Wuidcannotbeempty: "Wuid Cannot Be Empty.",
-        WUSnapshot: "WU Snapshot",
+        WuSnapShot: "WU SnapShot",
```
Comment (jeclrsg) [inline esp/src/src/nls/hpcc.ts:1133]: I'd change the casing on `WUSnapShot: "WU SnapShot"` to be `WUSnapShot: "WU Snapshot"`. Looks good otherwise

**Review by jeclrsg — CHANGES_REQUESTED** [review]

**Review by GordonSmith — APPROVED** [review]

**Review by jeclrsg — APPROVED** [review]

File: esp/src/src/nls/hpcc.ts [hunk 2]
```diff
@@ -1131,6 +1131,7 @@ export = {
         WUID: "WUID",
         Wuidcannotbeempty: "Wuid Cannot Be Empty.",
         WUSnapshot: "WU Snapshot",
+        WUSnapShot: "WU Snapshot"
```
Comment (ghalliday) [inline esp/src/src/nls/hpcc.ts:1134]: Should you also update the other language variants?

**Review by ghalliday — COMMENTED** [review]
One question.

File: esp/src/src/nls/hpcc.ts (hunk 2 above)
Comment (ghalliday) [reply esp/src/src/nls/hpcc.ts:1134]: @kunalaswani

Comment (kunalaswani) [reply esp/src/src/nls/hpcc.ts:1134]: Checking those files now to see if they need the change as well.

Comment (GordonSmith) [reply esp/src/src/nls/hpcc.ts:1134]: @kunalaswani - did you work out which version is preferred?

Comment (kunalaswani) [reply esp/src/src/nls/hpcc.ts:1134]: @ghalliday all the other files were already synced in their merges, only this one required this update. @GordonSmith both versions are being used, and the change to the pervious would require us to change 8 additional older files, it was discussed that we should retain the previous and this current version.

**Review by ghalliday — CHANGES_REQUESTED** [review]
See comment about es

File: esp/src/src/nls/hpcc.ts (hunk 2 above)
Comment (kunalaswani) [reply esp/src/src/nls/hpcc.ts:1134]: @ghalliday corrected, apologies on the miss in the ES file

**Review by ghalliday — APPROVED** [review]

**Review by ghalliday — CHANGES_REQUESTED** [review]
I just checked, and this isn't passing the code checks because it is missing a comma at the end of the line.  Was this tested?

**Review by ghalliday — APPROVED** [review]

**Pull request merged**